jwt.go

```diff
@@ -484,6 +490,11 @@ func (jwtPlugin *JwtPlugin) extractTokenFromCookie(request *http.Request) (strin
 	return cookie.Value, nil
 }
 
+func (jwtPlugin *JwtPlugin) extractTokenFromQuery(request *http.Request) (string, error) {
+	parameter := request.URL.Query().Get(jwtPlugin.jwtQueryKey)
```

Comment:
What happens if parameter not found in query ? Will it result in an error ?
This function should return proper error if query parameter is missing, as extract from header and extract from cookie do.

Reply:
Done.